src/components/Consent.js

```diff
@@ -23,14 +26,16 @@ export const Consent = () => {
       onKeyLeft: goto.termsOfUse,
       right: i18n('consent-softkeys-policy'),
       onKeyRight: goto.privacyPolicy,
+      onKeyArrowDown: scrollDown,
+      onKeyArrowUp: scrollUp,
       backspace: () => { window.exit() }
```

Comment:
this should be `onKeyBackspace`, i don't know why we use backspace

and it should also `window.close()`, not `window.exit()` ... 😨

Reply:
Good catch!

Reply:
Wait, backspace on the consent screen should indeed exit the app since you're not allowed to use it unless you explicitly agree to the terms.

Reply:
Yea I tested on device after https://github.com/wikimedia/wikipedia-kaios/pull/309/commits/a3ab443e803a18c1022dae8869af3e694b8f6f93 and I'm seeing that the app exits as expected, then on reopening the consent screen is displayed again